suite-native/module-passphrase/src/components/ConfirmOnDeviceBottomSheet.tsx

```diff
@@ -0,0 +1,60 @@
+import { BottomSheet, Box, Button, Text } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation } from '@suite-native/intl';
+
+import { DeviceTS3Svg } from '../assets/DeviceTS3Svg';
+
+const contentStyle = prepareNativeStyle(utils => ({
+    paddingTop: utils.spacings.small,
+    gap: utils.spacings.large,
+    alignItems: 'center',
+    paddingHorizontal: utils.spacings.small,
+}));
+
+const textContentStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.small,
+}));
+
+const buttonStyle = prepareNativeStyle(_ => ({
+    width: '100%',
+}));
+
+type ConfirmOnDeviceBottomSheetProps = {
+    isVisible: boolean;
+    onClose: (isVisible: boolean) => void;
```

Comment:
Check with Sheny, but I assume this component should only be closed by tapping on `Cancel` within this component or by device cancelation. Please remove this prop.

Reply:
da93c1a3af